src/state/atoms/notificationDrawerAtom.ts

```diff
@@ -0,0 +1,81 @@
+import { atom } from 'jotai';
+
+export type NotificationData = {
+  id: string;
+  title: string;
+  description: string;
+  read: boolean;
+  selected?: boolean;
+  source: string;
+  bundle: string;
+  created: string;
+};
+
+export type NotificationsPayload = {
+  data: NotificationData;
+  source: string;
+  // cloud events sub protocol metadata
+  datacontenttype: string;
+  specversion: string;
+  // a type field used to identify message purpose
+  type: string;
+  time: string;
+};
+
+export const notificationDrawerExpandedAtom = atom(false);
+export const notificationDrawerDataAtom = atom<NotificationData[]>([]);
+export const notificationDrawerCountAtom = atom(0);
```

Comment:
I think the initial data should be passed from chrome as well as props. The notifications module should only consume them.